src/models/SessionReturnRecord.ts

```diff
@@ -0,0 +1,67 @@
+import { ReturnSQSEvent } from "./ReturnSQSEvent";
+import { Constants } from "../utils/Constants";
+
+export interface NamePart {
+	type: string;
+	value: string;
+}
+
+export interface PersonName {
+	nameParts: NamePart[];
+}
+export class SessionReturnRecord {
+	constructor(data: ReturnSQSEvent, expiresOn: number) {
+		this.userId = data.user.user_id;
+		switch (data.event_name) {
+			case Constants.AUTH_IPV_AUTHORISATION_REQUESTED:{
+				this.clientName = data.client_id!;
+				this.redirectUri = data.component_id!;
+				this.userId = data.user.user_id;
```

Comment:
we can remove this one as doing that earlier anyways

Reply:
Done